source/camera/basic/capture_from_file.py

```diff
@@ -8,15 +13,28 @@
 
 def _main():
     app = zivid.Application()
-    camera = app.create_file_camera(Path() / get_sample_data_path() / "FileCameraZividOne.zfc")
 
+    # The file_camera file is in Zivid Sample Data. See instructions in README.md
+    file_camera = Path() / get_sample_data_path() / "FileCameraZividOne.zfc"
+
+    print(f"Creating virtual camera using file: {file_camera}")
+    camera = app.create_file_camera(file_camera)
+
+    print("Configuring settings")
     settings = zivid.Settings()
     settings.acquisitions.append(zivid.Settings.Acquisition())
-    settings.acquisitions[0].aperture = 5.66
-    settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=8333)
+    settings.processing.filters.smoothing.gaussian.enabled = True
+    settings.processing.filters.smoothing.gaussian.sigma = 1.5
+    settings.processing.filters.reflection.removal.enabled = True
+    settings.processing.color.balance.red = 1.0
+    settings.processing.color.balance.green = 1.0
+    settings.processing.color.balance.blue = 1.0
```

Comment:
Agreed. The file camera does not respond to Acquisition settings, but it does respond to filters. Less confusing now.